cmd/juju/commands/debughooks_test.go

```diff
@@ -80,10 +76,20 @@ var debugHooksTests = []struct {
 	expected: &argsSpec{
 		hostKeyChecking: "yes",
 		knownHosts:      "0",
-		enablePty:       true,
 		withProxy:       true,
 		argsMatch:       `ubuntu@0\.(private|public|1\.2\.3) sudo .+`, // can be any of the 3
 	},
+}, {
+	info:        "pty enabled",
+	args:        []string{"--pty", "mysql/0"},
```

Comment:
> I see that you're testing an explicit --pty, though the documentation says to use --pty=true and --pty=false. I know how the gnuflag parser works, but I wonder if it would be good to be testing the way we're documenting it should work?

sure, done

> It does seem like it would be good to test explicit behavior. So maybe just adding 1 more step with a --pty=False to show that the user has control?

already tested in the ssh command, I don't think it's worth testing here too